tests/conftest.py

```diff
@@ -246,6 +259,17 @@ def collect(self):
         args.append(str(self.path.parent))
         yield Repo2DockerTest.from_parent(self, name="build", args=args)
 
+        # If external-verify exists it should be run on the host
+        external_verify_script = self.path.parent / "external-verify"
+        if external_verify_script.exists():
+            yield Repo2DockerTest.from_parent(
+                self,
+                name=self.path.name,
```

Comment:
This sets the test name for the external-verify (currently 'verify'):

```suggestion
                name=external_verify_script.name,
```